src/components/app/ProfileSharing.js

```diff
@@ -93,9 +99,12 @@ class ProfileSharingCompositeButton extends PureComponent<
   _permalinkButton: ButtonWithPanel | null;
   _uploadErrorButton: ButtonWithPanel | null;
   _permalinkTextField: HTMLInputElement | null;
+  _secondShareButton: ButtonWithPanel | null;
```

Comment:
I am not sure about the naming here as `second` can also mean to support something. @julienw whats your opinion on this?

Reply:
The naming was towards using `second` in that context in general as we have more vars with that.

Reply:
+1 on secondary :)

Reply:
Yeah, I guess secondary would be better but @julienw is right, we don't need that property at all :)

Reply:
as ola said, this comment was also about all the other uses of "second" in the same file :)

Reply:
Oh, okay, will change others too then :)